litho-widget/src/main/java/com/facebook/litho/widget/RecyclerBinder.java

```diff
@@ -305,6 +306,22 @@ public Builder hasDynamicItemHeight(boolean hasDynamicItemHeight) {
       return this;
     }
 
+    /**
+     * Enable setting custom viewTypes on {@link ViewRenderInfo}s.
+     *
+     * After this is set, all {@link ViewRenderInfo}s must be built with a custom viewType through
+     * {@link ViewRenderInfo.Builder#customViewType(int)}.
```

Comment:
Can you add something like  ", otherwise it will throw exception when items are inserted to binder" to the end?